### PR TITLE
[YOLO] moved tests

### PR DESCRIPTION
**Pull request by eric-s-s**
### Summary

This moves the unit tests inside the drum folder.  otherwise they don't get run.  once it is complete, you should be able to find the test in: https://jenkins.hq.datarobot.com/job/Custom_Models_Templates_CMRunner_Integration_Tests/2431/console

under tests/drum/unit

the tests were skipped before

### CODE REVIEW

**Comment by eric-s-s** [comment]
the tests are not skipped now 
image

File: tests/drum/unit/test_typeschema_validation.py [hunk 1]
```diff
@@ -86,8 +88,6 @@ def output_requirements_yaml(
 
 
 class TestSchemaValidator:
-    tests_data_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "testdata"))
```
Comment (eric-s-s) [inline tests/drum/unit/test_typeschema_validation.py:89]: did nothing. removing to no ill effect

**Comment by yakov-g** [comment]
number of test cases in this pr goes from 613 to 790, is it expected?

**Comment by eric-s-s** [comment]
> number of test cases in this pr goes from 613 to 790, is it expected?

@yakov-g yes.  it's actually running the unit tests now.

**Review by 4dahalibut — APPROVED** [review]

**Review by scottp-ml — APPROVED** [review]

**Pull request merged**